### ISSUE TITLE
Repeating sets: Action Buttons Do Not Announce Resulting Action with Screen Reader 

### ISSUE THREAD

**Issue by samsadasivan** [cds-snc/platform-forms-client#5840]
**_Specs: Windows
Screen Readers Used: JAWS, NVDA
Browsers Tested: Google Chrome, Microsoft Edge, Internet Explorer_**

**Components:** 
“Add another team member” & “Remove team member” 

**Observation:** 
1. Screen reader announces the button label but does not announce the outcome of the action.  

2. No feedback is provided, leaving users unaware of whether the action was successful 

3. No live region or update announcement is triggered after the action. 

**_Compounded Severity: High_**

**Recommendation:**  
When activating the “Add Another Team Member” or “Remove Team Member” button, the screen reader should announce the result of the action (e.g., "Team member added", "Team member removed"). 

Users should receive immediate confirmation or status update when an action is performed. 

1. Announce Action Outcomes Clearly 

Ensure the result of the action (e.g., member added/removed) is announced via the screen reader immediately after the button is activated. 

2. Ensure Semantic Button Roles Are Present 

Confirm that both “Add” and “Remove” elements are using the semantic <button> element or explicitly set role="button".

**Comment by thiessenp-cds** [comment]
This has since been fixed.
e.g. when adding or removing a repeating set, the operation is announced


<img width="1234" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL2ErQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8171422f0322bebd67ed2a8c25c672ae0af6b27a/Screenshot%202025-07-16%20at%202.50.55%E2%80%AFPM.png" alt="Screenshot 2025-07-16 at 2.50.55 PM.png" />
